providers/azure/resources/keyvault.go

```diff
@@ -211,6 +211,71 @@ func (a *mqlAzureSubscriptionKeyVaultServiceVault) keys() ([]interface{}, error)
 	return res, nil
 }
 
+func (a *mqlAzureSubscriptionKeyVaultServiceKeyAutorotation) keyName() (string, error) {
+	id := a.Kid.Data
+	kvid, err := parseKeyVaultId(id)
+	if err != nil {
+		return "", err
+	}
+
+	return kvid.Name, nil
```

Comment:
is the name guaranteed unique? if not, could just use the id itself here

Reply:
yes, it is always unique!